src/azul/chalice.py

```diff
@@ -404,11 +407,23 @@ def _log_request(self):
                 'query': self.current_request.query_params,
                 'headers': self.current_request.headers
             }
-            log.info('Received %s request for %r, with %s.',
+            log.info('Received %s request for %r, with %s',
                      context['httpMethod'],
                      context['path'],
                      json.dumps(request_info, cls=self._LogJSONEncoder))
 
+            body = self.current_request.json_body
+            if body is None:
+                len_msg = ''
+            elif config.debug == 2:
```

Comment:
```suggestion
            elif config.debug > 1:
```

If we ever add a fourth level m we would want this condition to be True there as well.